.github/workflows/deploy_website.yml

```diff
@@ -0,0 +1,53 @@
+name: Deploy Documentation to GitHub Pages
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
```

Comment:
You should remove this, for 2 reasons.
1. You wouldn't actually want PRs to deploy your website, that's a really bad attack vector
2. The built-in token doesn't have permissions in the pull request context

Reply:
Will do, I added it to test the workflow run.

Reply:
To test this workflow would I just land and then wait for a diff to be pushed to main?